src/Transformations.jl

```diff
@@ -215,56 +213,14 @@ end
 function transform!(tn::AbstractTensorNetwork, config::ColumnReduction)
     skip_inds = isempty(config.skip) ? inds(tn, set = :open) : config.skip
```

Comment:
```suggestion
    skip_inds = config.skip
```
Couldn't we just put this here in the `ColumnReduction` struct definition? The user may want to actually have the open indices reduced.

Reply:
Nope, because at the struct definition you don't have the tensor network yet.